cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/Person.java

```diff
@@ -239,12 +238,11 @@ private Map<String, String> convertTypedValuesToMap(Set<TypedValue> typedValueSe
      * Converts NVA formatted employments to Cristin formatted employments.
      */
     public static List<CristinPersonEmployment> mapEmploymentsToCristinEmployments(Set<Employment> employments) {
-        if (isNull(employments)) {
-            return null;
-        }
-        return new ArrayList<>(employments).stream()
-                   .map(Employment::toCristinEmployment)
-                   .collect(Collectors.toList());
+        return Optional.ofNullable(employments)
+            .orElse(Collections.emptySet())
+            .stream()
+            .map(Employment::toCristinEmployment)
+            .toList();
```

Comment:
Returning null here is intended as we dont want to include it at all if empty. Im not sure if it causes errors, but i remember working on it.

We also do not return an empty list of employments to unauthorized clients, therefore we set it to null so it is not returned in json. this field should only be available to authorized users, and im also not sure how Cristin will react when POST'ing an empty list

Reply:
I think the safest is to leave it for now